Services/Awareness/classes/class.WidgetManager.php

```diff
@@ -99,14 +100,14 @@ public function notifyOnNewOnlineContacts() : void
             return;
         }
 
-        $notification = new \ilNotificationConfig('who_is_online');
+        $notification = new ilNotificationConfig('who_is_online');
```

Comment:
A suggestion: It would be great to get this instance from a factory that is somehow available through the DIC.

Reply:
Good point @alex40724

Reply:
I agree. But i assume this would be better fitting in a own PR.